jp/stripStopWords.js

```diff
@@ -0,0 +1,26 @@
+module.exports={
+	strip_stopwords: function(languageCode, inputText){
+		var fileSystem = require('fs');
+		var validLanguageCodes = fileSystem.readFileSync('../data/list.txt').toString().split('\n');
+		var languageCodeValid = validLanguageCodes.indexOf(languageCode);
+
+		if (languageCodeValid === -1){
+			return "Error: Not a valid language code"
+		}
+
+		var stopWords = fileSystem.readFileSync('../data/' + languageCode + '.txt').toString().split('\n');
+		var splitInput = inputText.toString().split(" ");
+		var finalString= "";
+
+		for (word in splitInput){
```

Comment:
This is very minor stuff. Due to the fact that JavaScript has a strange for-in looping, we could make this 'word' variable clearer by naming it as 'index' or 'i'.

Reply:
Great point! I will adjust that to make it more clear!